src/buttons.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { useState } from "react";
+import swal from "sweetalert";
+
+function TaskForm() {
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+
+  const showAlert = () => {
+    swal({
+      title: "confirm",
+      text: "Are you sure you want to delete this task?",
+      icon: "waring",
+      button: ["acept", "cancel"],
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+
+  return (
+    <div className="ma-w-md mx-auto">
+      <h1 className="2xl font-bold text white mb-3">Add new task</h1>
+      <form onSubmit={handleSubmit} className="bg-slate-800 p-10 mb-4">
+        <input
+          minLength={3}
+          maxLength={30}
+          placeholder="write"
+          onChange={(e) => setTitle(e.target.value)}

```

Comment:
Create a function instead of using an arrow function directly here

For example, in the form, you have inside the onSubmit event a function called “handleSubmit”